### DISCUSSION TITLE
Arrow position in Landscape mode

### DISCUSSION THREAD

**Discussion by Petak73** [organicmaps/organicmaps#8548] [Ideas]
Hi,

Two suggestions:
- move the arrow in landscape nav mode down and to the right (see attached screenshot) in order to allow more field of view ahead.
- provide an option to remove the street name bar on top for the same reason - more field of view ahead. Such option would be useful in portrait view as well.

As a side note - is it possible to make the navigations screen completely fullscreen (or immersive as it used to be called) by hiding the navigation bar at the bottom and the status bar on top or is this dependant on the OS? Again, it doesn't bother me much in portrait view but takes quite a lot of space in landscape. I know it is transparent but the UI elements of the app (direction arrow, info bar etc.) can not go on top of it so it still takes valuable screen real estate.

![Screenshot_20240624-075416](https://github.com/organicmaps/organicmaps/assets/173553515/9a809149-59c4-449c-b04b-273e2c76015a)

**Comment by biodranik** [comment]
This is a planned improvement, there were several related issues about it. Please check existing issues and create new ones if necessary.

**Reply by Petak73** [reply]
That's great to know. Thank you.
If there are issues moving the arrow to the side you can make the nav bar (speed, est time and distance) vertical. The field of view in landscape mode is limited only vertically, so moving everything to the sides will make it both symmetrical/nicer looking and more functional allowing to view further ahead.
